### PR TITLE
dependabot: Resolve updates

### PR DESCRIPTION
**Pull request by phlax**
Commit Message: dependabot: Resolve updates
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: api/envoy/extensions/filters/http/cache/v3alpha/cache.proto [hunk 1]
```diff
@@ -19,7 +19,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: HTTP Cache Filter]
 
-// [#extension: envoy.filters.http.cache.simple_http_cache]
+// [#extension: envoy.filters.http.cache]
```
Comment (phlax) [inline api/envoy/extensions/filters/http/cache/v3alpha/cache.proto:22]: @htuch not sure if this is correct 

i never really understood why but the cache proto is defined in `/source` and in the api its defined as simple_http_cache

if we dont build the v2 protos `envoy.filters.http.cache` is no longer available and building fails

Comment (phlax) [reply api/envoy/extensions/filters/http/cache/v3alpha/cache.proto:22]: oops wrong pr

**Comment by phlax** [comment]
apologies  for noise - i added commit to wrong pr

**Review by moderation — APPROVED** [review]

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**